lib/codegen/fromcto/vocabulary/vocabularyvisitor.js

```diff
@@ -179,10 +180,16 @@ class VocabularyVisitor {
     visitProperty(property, parameters) {
         const declarationName = property.getParent().getName();
         const propertyName = property.getName();
+    
+        // Skip generating vocabulary for system property names
+        if (ModelUtil.isSystemProperty(propertyName) == true) {
```

Comment:
` if (ModelUtil.isSystemProperty(propertyName))` will also work.
Also any reason why `==` instead of `===` ?